base-notebook/Dockerfile.ppc64le

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) IBM Corporation 2016 
```

Comment:
OK. I don't fully understand the benefit of the patch file if it's only used to generate a file that's already there, but we can refine later.

One last question: is there a reason that you have changed the license of this file to be different from the rest? Is it possible to leave the license as BSD so that it's consistent with the rest of the repository?